cypress/e2e/tests/footer.cy.js

```diff
@@ -0,0 +1,220 @@
+context('Footer Get Started', () => {
+  beforeEach(() => {
+    cy.visit('/')
+    cy.get('div')
```

Comment:
For key sections of the page, we should have `id` or unique `class` attributes so that we can leverage it in our Tests making it somewhat faster Test run. In this case, we're traversing through all `div(s)` and checking if it has desired text.